app/models/manageiq/providers/container_manager.rb

```diff
@@ -4,15 +4,16 @@ class ContainerManager < BaseManager
     include SupportsFeatureMixin
 
     has_many :container_nodes, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :container_groups, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :container_groups, -> { active }, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_services, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_replicators, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :containers, :foreign_key => :ems_id
-    has_many :container_projects, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :containers, -> { active }, :foreign_key => :ems_id
+    has_many :container_definitions, -> { active }, :foreign_key => :ems_id
```

Comment:
@cben you've defined ```has_many :container_definitions, :through => :container_groups``` in the https://github.com/ManageIQ/manageiq/pull/15298/files

but there is :ems_id on the container_definitions